rsocket/RSocketServer.cpp

```diff
@@ -128,7 +128,10 @@ void RSocketServer::onRSocketSetup(
     throw result.error();
   }
   auto connectionParams = result.value();
-  CHECK(connectionParams.responder);
+  if (!connectionParams.responder) {
+    LOG(ERROR) << "Received invalid Responder. Dropping connection";
```

Comment:
If the server-application does not want to accept the connection, it should ideally return a RSocketException.  This case is for erroneous applications which return a NULL responder by mistake.  I did not want to terminate the server in those cases.  We can reconsider this later if it turns to be used as a feature (I hope not).